k-distribution/src/main/scripts/bin/kparse

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+set -u
+
+export PATH="$(cd "$(dirname "$0")"; pwd):$PATH"
+
+# initialize flags
+ARGV=()
```

Comment:
Bash associative arrays on MacOS issue.

Reply:
This isn't an associative array. Just a regular array, which is fine.